### PR TITLE
MOSIP-34308

### PR DESCRIPTION
**Pull request by likhitharl**
updated description for endpoint 
GetRegistrationCenterByHierarchyLevelAndTextPaginated

### CODE REVIEW

File: apitest/src/main/resources/resident/GetRemainingupdatecountbyIndividualIdRequest/GetRemainingupdatecountbyIndividualIdRequest.yml [hunk 1]
```diff
@@ -52,7 +55,8 @@ GetRemainingupdatecountbyIndividualIdRequest:
 
    Resident_GetRemainingupdatecountbyIndividualIdRequest_Invalid_Token:
       endPoint: /idrepository/v1/identity/{individualId}/update-counts?idType={idType}&attribute_list={attribute_list}
-      role: partner
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/resident/GetRemainingupdatecountbyIndividualIdRequest/GetRemainingupdatecountbyIndividualIdRequest.yml]: Role should not be changed here

File: apitest/src/main/resources/resident/SendOtpForContactDet/SendOtpForContactDet.yml [hunk 2]
```diff
@@ -272,9 +286,10 @@ SendOtpForContactDet:
 
    Resident_SendOtpForContactDet_vid_Invalid_Transaction_Id_Neg:
       endPoint: /resident/v1/contact-details/send-otp
+      description: Request otp when invalid transactionid is given and logged in uisng VID      
       role: residentNewVid
       restMethod: post
-      inputTemplate: resident/SendOtpForContactDet/SendOtpForContactDet
+      inputTemplate: resident/Sen	dOtpForContactDet/SendOtpForContactDet
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/resident/SendOtpForContactDet/SendOtpForContactDet.yml]: Remove the spaces added and make it as unchanged

File: apitest/testNgXmlFiles/residentApi.xml [hunk 3]
```diff
@@ -24,13 +24,13 @@
 			<class name="io.mosip.testrig.apirig.testscripts.PostWithAutogenIdWithOtpGenerate" />
 		</classes>
```
Comment (mohanachandran-s) [inline apitest/testNgXmlFiles/residentApi.xml]: xml file should be unchanged. Remove the modifications done

File: apitest/testNgXmlFiles/residentApi.xml [hunk 4]
```diff
@@ -340,29 +340,38 @@
 			<class name="io.mosip.testrig.apirig.testscripts.SimplePost" />
 		</classes>
 	</test>
-	<test name="GetIdentityMapping">
+	<test name="GetPartnersByPartnerType">
```
Comment (mohanachandran-s) [inline apitest/testNgXmlFiles/residentApi.xml]: why below 9 lines are added newly? Can you replace it with upstream revision if it is not intended

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

File: apitest/src/main/resources/resident/GetRemainingupdatecountbyIndividualIdRequest/GetRemainingupdatecountbyIndividualIdRequest.yml (hunk 1 above)
Comment (likhitharl) [reply apitest/src/main/resources/resident/GetRemainingupdatecountbyIndividualIdRequest/GetRemainingupdatecountbyIndividualIdRequest.yml]: @mohanachandran-s Role is Resident only,

File: apitest/testNgXmlFiles/residentApi.xml (hunk 3 above)
Comment (likhitharl) [reply apitest/testNgXmlFiles/residentApi.xml]: changes reverted

File: apitest/testNgXmlFiles/residentApi.xml (hunk 4 above)
Comment (likhitharl) [reply apitest/testNgXmlFiles/residentApi.xml]: Replaced with upstream revision

**Review by mohanachandran-s — APPROVED** [review]

**Review by lsivanand — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**